doc/news/changes/minor/20190807ManaswineeBezbaruah

```diff
@@ -0,0 +1,3 @@
+The original documentation of enclosed_hyper_cube said 'first two parameters' in the second line, which was a bit unclear. 
+I made that more specific. The coloring scheme was also unclear, especially with the use of bitwise OR as a verb.
+The moebius function had a pi outside of math mode, not a huge error.
```

Comment:
Could you please add your name and the date to the end of this file? Try looking at one of the other changelog entries to see what the standard format is

Reply:
Done!